### PR TITLE
TRACING-3319: Migrate upstream Tempo operator upstream docs to Grafana

### PR DESCRIPTION
**Pull request by pavolloffay**
Version(s):

OCP 4.10 and above

Issue:

https://issues.redhat.com/browse/TRACING-3319

Link to docs preview:

https://63048--docspreview.netlify.app/

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/distr-tracing-rn-technology-preview.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ For services that have not sent traces within the last 15 minutes, those traces
 
 Expanded support for the Tempo Operator is planned for future releases of {DTProductName}.
 Possible additional features might include support for TLS authentication, multitenancy, and multiple clusters.
-For more information about the Tempo Operator, see link:https://tempo-operator.netlify.app[the Tempo community documentation].
+For more information about the Tempo Operator, see link:https://grafana.com/docs/tempo/latest/setup/operator/[the Tempo community documentation].
```
Comment (max-cx) [inline modules/distr-tracing-rn-technology-preview.adoc:38]: If I understand correctly, it's (as a matter of fact) the documentation for `Community Tempo Operator`. If so, then I suggest changing it a bit:
```suggestion
For more information about the Tempo Operator, see the documentation for the link:https://grafana.com/docs/tempo/latest/setup/operator/[Community Tempo Operator].
```
image

Comment (pavolloffay) [reply modules/distr-tracing-rn-technology-preview.adoc]: Change applied

**Review by max-cx — APPROVED** [review]

**Review by jldohmann — APPROVED** [review]
LGTM

for future PRs, it's really helpful to have a direct link to the location in the docs where your edits appear because reviewers don't always know where to look in such a massive doc set. For example, for this PR the preview link would be: https://63048--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes#red-hat-openshift-distributed-tracing-2-8-0-technology-preview

**Pull request merged**